ipatests/test_integration/test_caless.py

```diff
@@ -1574,10 +1574,11 @@ def test_server_ipa_ca_install(self):
 
         ca_master = tasks.install_ca(self.master)
         assert ca_master.returncode == 0
-        cert_update_master = self.master.run_command(['ipa-certupdate'])
-        assert cert_update_master.returncode == 0
-        cert_update_replica = self.replicas[0].run_command(['ipa-certupdate'])
-        assert cert_update_replica.returncode == 0
+        # We are not calling ipa-certupdate on replica here since the next step
+        # installs CA clone there.
+
+        ca_show = self.master.run_command(['ipa', 'ca-show', 'ipa'])
+        assert ca_show.returncode == 0
```

Comment:
Could you please check the output of the command to verify that it prints the correct CA?

``assert ca_show.returncode == 0`` shouldn't be necessary because ``run_command`` raises an exception by default.

Reply:
sure, done...